templater/templater_test.go

```diff
@@ -34,4 +34,33 @@ var _ = Describe("Templater", func() {
 			Expect(content).To(ContainSubstring(`Responsible Role: Amazon Elastic Compute Cloud: AWS Staff`))
 		})
 	})
+	Describe("DiffSSP", func() {
+		It("should warn the user if the current SSP contains a responsible role that conflicts with the " +
+			"responsbile role in the YAML", func() {
+
+			By("Loading the SSP with the Responsible Role being 'OpenControl Role Placeholder' " +
+				"for Control 'AC-2 (1)'")
```

Comment:
I just mean that if the tests are long enough where they feel necessary, then helpers could be used to make them shorter and more clear.